### ISSUE TITLE
Merging Not Working

### ISSUE THREAD

**Issue by swift-ben** [SFDO-Community/declarative-lookup-rollup-summaries#936]
I have a rollup that calculates data from a child object of Contact

I followed the instructions in [these release notes](https://andyinthecloud.com/2016/06/19/declarative-rollup-tool-summer-release/), making sure my trigger was set to support parent merging and that RollupJobs was scheduled. 

Using both the NPSP Merge page and merging via Apex, the rollups didn't recalculate. When I ran them manually from the DLRS Manage Rollup page, they calculated fine. Is there something else I'm missing?

**Comment by afawcett** [comment]
@swift-ben is the Active field on the rollup set?

**Comment by swift-ben** [comment]
@afawcett yep, it's active and the Rollup works when I create the duplicates for testing before the merge (and it works when I run calculate after the merge). It just doesn't work immediately after the merge without forcing the calculate.

**Comment by aheber** [comment]
Merge actions will not process in realtime, even if the rollup is set to realtime. Ensure your have the RollupJob scheduled (very easing using the v2.21 "New Wizard" that has a graphical schedule manager). That will process the merge rollups on delay but they should get updated.
